### PR TITLE
pt-182087803 Upgrade node; update regex for log filtering

### PR DESCRIPTION
**Pull request by joelwalden**
Ticket: https://www.pivotaltracker.com/story/show/182087803

Test: Regex has been updated per ticket: https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/functions/serverlessrepo-sumologic-SumoLogGroupLambdaConnec-1MMCQVW6C6OEO?tab=configure

### CODE REVIEW

File: loggroup-lambda-connector/sam/packaged.yaml [hunk 1]
```diff
@@ -21,9 +21,9 @@ Metadata:
     - serverless
     - loggroups
     - cloudwatch
-    LicenseUrl: s3://appdevstore/LoggroupConnector/v1.0.5/6092dd6c323e33634657102f570628e0
+    LicenseUrl: s3://kbs-reporting-db-production/LoggroupConnector/v1.0.4/6092dd6c323e33634657102f570628e0
```
Comment (girwin-kbs) [inline loggroup-lambda-connector/sam/packaged.yaml:24]: Can you put this in a different bucket? The log group connector is nothing to do with reporting db.

File: loggroup-lambda-connector/sam/packaged.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ Parameters:
     - Kinesis
   DestinationArnValue:
     Type: String
-    Default: arn:aws:lambda:us-east-1:123456789000:function:TestLambda
+    Default: arn:aws:lambda:us-west-2:123456789000:function:TestLambda
```
Comment (girwin-kbs) [inline loggroup-lambda-connector/sam/packaged.yaml:42]: This is not the KBS AWS account is it? I would expect the arn to start with `arn:aws:lambda:us-west-2:529513974030`

File: loggroup-lambda-connector/sam/sam_package.sh [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
 
-if [ "$AWS_PROFILE" == "prod" ]
-then
-    SAM_S3_BUCKET="appdevstore"
-    AWS_REGION="us-east-1"
-else
-    SAM_S3_BUCKET="cf-templates-5d0x5unchag-us-east-2"
-    AWS_REGION="us-east-2"
-fi
+SAM_S3_BUCKET="kbs-reporting-db-production"
```
Comment (girwin-kbs) [inline loggroup-lambda-connector/sam/sam_package.sh:3]: different bucket please

File: loggroup-lambda-connector/sam/template.yaml [hunk 4]
```diff
@@ -41,7 +41,7 @@ Parameters:
 
   DestinationArnValue:
     Type: String
-    Default: "arn:aws:lambda:us-east-1:123456789000:function:TestLambda"
+    Default: "arn:aws:lambda:us-west-2:123456789000:function:TestLambda"
```
Comment (girwin-kbs) [inline loggroup-lambda-connector/sam/template.yaml:44]: This is not the KBS AWS account is it? I would expect the arn to start with arn:aws:lambda:us-west-2:529513974030

**Review by girwin-kbs — CHANGES_REQUESTED** [review]

File: loggroup-lambda-connector/sam/packaged.yaml (hunk 2 above)
Comment (joelwalden) [reply loggroup-lambda-connector/sam/packaged.yaml:42]: This is overridden anyway, but I'll change the default to use our account.

**Comment by joelwalden** [comment]
@girwin-kbs Updated the Lambda ARN; this is the working version

**Review by dwarren-kbs — APPROVED** [review]

**Pull request merged**